### DISCUSSION TITLE
Meaning of "backup interval" in 'times_setup.dat' and input of rainfall data

### DISCUSSION THREAD

**Discussion by jennykupzig** [SynxFlow/SynxFlow#18] [Q&A] [answered]
After reading the docs from synxflow and HiPIMS it is not clear for me what the "backup interval" means. Could you please elaborate a bit more on the four values and the link e.g., to the rainfall data (precipitation_source_all.dat). 

For example, how sboth files should look like, when I want to run a model for 2 hours (and I need only the maximal water depth and velocity ocurring within this time period) and I want to use areal precipitation that is e.g., 50mm in the first hour (and then zero).

**Comment by xiaxilin** [comment, marked as answer]
Hi, this is a function that writes the output as the same format of the inputs, which can be then used as the initial conditions. But with the new Python integration, this is less useful now.
